src/views/penalty-reference.njk

```diff
@@ -0,0 +1,121 @@
+{% extends "layout.njk" %}
+
+{% from "govuk/components/back-link/macro.njk"  import govukBackLink %}
+{% from "govuk/components/input/macro.njk"      import govukInput %}
+{% from "govuk/components/details/macro.njk"    import govukDetails %}
+{% from "govuk/components/button/macro.njk"     import govukButton %}
+
+
+
+{% block pageTitle %}
+  {% if errorList.length > 0 %}
+    Error:
+  {% endif %}
+  Tell us the penalty details
+{% endblock %}
+
+{% block beforeContent %}
+  {{ govukPhaseBanner({
+    tag: {
+      text: "alpha"
+    },
+    html: 'This is a new service – your <a class="govuk-link" href="#">feedback</a> will help us to improve it.'
+    }) 
+  }}
+  {{ 
```

Comment:
I realize this was actually my work, but looking back at it maybe we should just refactor it so that it looks like:
{{
   govukPhaseBanner {{
    ..... 
   })
}}
Damian had made the suggestion to me in lines 38 to 49 of this same file and it looks much better